python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -1035,6 +1035,35 @@ def mean_pattern() -> tvm.relay.dataflow_pattern.DFPattern:
     return pattern
 
 
+class SigmoidParams:
```

Comment:
Should we also create a subclass for this and `TanhParams`?

Reply:
Done